app/javascript/hyacinth_v1/components/digital_objects/rights/rights_form/ItemRightsForm.js

```diff
@@ -36,7 +36,10 @@ function ItemRightsForm(props) {
 
   const [rights, setRights] = useHash(mergeDefaultValues(fieldConfiguration, initialRights));
 
-  const [updateRights, { error: updateError }] = useMutation(updateRightsMutation);
+  const [updateRights, { data: updateData }] = useMutation(updateRightsMutation);
+
+  // One day, maybe enable optionalChaining JS feature in babel to simplify lines like the one below.
+  const userErrors = (updateData && updateData.updateRights && updateData.updateRights.userErrors) || [];
```

Comment:
Same note here about wanting to include `error: updateError` and still show it in with `GraphQLErrors` component (which only renders if updateErrors are actually present).